lib/twingly/url.rb

```diff
@@ -100,6 +100,10 @@ def tld
       public_suffix_domain.tld
     end
 
+    def ttld
```

Comment:
What would the comment say? Wouldn't it be better to call the method `true_tld` then?